### PR TITLE
Add Login Feature

### PR DESCRIPTION
**Pull request by ctola91**
Create generic login feature for automation test

### CODE REVIEW

**Comment by ctola91** [comment]
image

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Login/Login.java [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
 public class Login extends BasePage {
-    @FindBy(name = "email")
+    @FindBy(name = "username")
     private WebElement usernameInputField;
 
     @FindBy(name = "password")
     private WebElement passwordInputField;
 
-    @FindBy(css = ".btn-primary.btn-block")
+    @FindBy(tagName = "button")
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Login/Login.java]: use unique locator

**Review by marcelogaray — CHANGES_REQUESTED** [review]
review observations

File: src/test/resources/features/Home/Home.feature [hunk 2]
```diff
@@ -1,9 +1,8 @@
 # Login feature review the behavior in thie feature
 Feature: Home
-
   Background:
     Given 'Login' page is loaded
     And set my credentials on 'Login' page
 
   Scenario: Incident page is displayed once click into 'Incident' link
-    Given Click on 'Incident' link
+    Given 'Home' Page is loaded
```
Comment (marcelogaray) [inline src/test/resources/features/Home/Home.feature]: https://github.com/marcelogaray/auto-GA-v03/pull/14
Example
Given Im authenticated on 'SMARTHOUSE' page with properly credentials

**Review by marcelogaray — CHANGES_REQUESTED** [review]
review observation

**Comment by marcelogaray** [comment]
Ok, thank you by the moment I will close it, please send me pull request atomics.

**Pull request closed without merging**